app/src/processing/app/Mode.java

```diff
@@ -565,77 +565,6 @@ public void actionPerformed(ActionEvent e) {
     }
   }
 
-
```

Comment:
I fail to see the point of keeping so much commented out code in the codebase. It is pretty easy nowadays to find deleted code with version control systems. It's a bad practice in almost every code style guide.

If there is a reason, I think it should be documented so that new contributors don't waste time cleaning up commented code.